src/main/java/org/nibor/autolink/LinkExtractor.java

```diff
@@ -88,8 +93,9 @@ public Builder emailDomainMustHaveDot(boolean emailDomainMustHaveDot) {
          */
         public LinkExtractor build() {
             UrlScanner urlScanner = linkTypes.contains(LinkType.URL) ? new UrlScanner() : null;
+            WwwUrlScanner wwwScanner = linkTypes.contains(LinkType.URL) ? new WwwUrlScanner() : null;
```

Comment:
- [ ] Can you please add a new `LinkType.WWW` for this instead? Some users of the library might want to opt out of www scanning but keep normal URL scanning.